networks/suzuka/suzuka-config/src/lib.rs

```diff
@@ -1,36 +1,44 @@
+use thiserror::Error;
+
+#[derive(Error, Debug)]
+pub enum ConfigError {
+	#[error("Could not read the Environment variables")]
+	EnvReadError,
+	#[error("Could not write the Environment variables")]
+	EnvWriteError,
+	#[error("Could not write the Bash export string")]
```

Comment:
I feel like we probably want to report which variables causes the error.

Reply:
I agree, added them.